Core/src/org/sleuthkit/autopsy/ingest/runIngestModuleWizard/RunIngestModulesAction.java

```diff
@@ -103,35 +117,9 @@ public void actionPerformed(ActionEvent e) {
         }
     }
 
-    /**
-     * Display any warnings that the ingestJobSettings have.
-     *
-     * @param ingestJobSettings
-     */
-    private static void showWarnings(IngestJobSettings ingestJobSettings) {
-        List<String> warnings = ingestJobSettings.getWarnings();
-        if (warnings.isEmpty() == false) {
-            StringBuilder warningMessage = new StringBuilder(1024);
-            for (String warning : warnings) {
-                warningMessage.append(warning).append("\n");
-            }
-            JOptionPane.showMessageDialog(null, warningMessage.toString());
-        }
-    }
-
-    @Override
-    public void performAction() {
-        actionPerformed(null);
-    }
-
-    @Override
-    public String getName() {
-        return Bundle.RunIngestModulesAction_name();
-    }
 
     @Override
-    public HelpCtx getHelpCtx() {
-        return HelpCtx.DEFAULT_HELP;
+    public Object clone() throws CloneNotSupportedException {
+        return super.clone(); //To change body of generated methods, choose Tools | Templates.
```

Comment:
Please change this implementation to throw CloneNotSupportedException. This implementation would result in sharing references to the  List field and the items in the list among clones. A deep copy approach  like that used by the superclass would work, but it's not worth the effort. Thanks!